ttml.go

```diff
@@ -421,6 +432,15 @@ func ReadFromTTML(i io.Reader) (o *Subtitles, err error) {
 			}
 
 		}
+		//if multiple items in `l` are present without new line then adding space at end so it can become readable after concatenation
+		if len(l.Items) > 0 {
```

Comment:
Can you provide an example `ttml` that was having issue with that bit of code?

Reply:
I used testdata/example-in.ttml

After adding this fix, SRT, and STL output added this additional space while joining the texts, so I added code to replace extra space while joining the Text for SRT and STL.